tmkms.toml.example

```diff
@@ -17,3 +17,5 @@ adapter = { type = "usb" }
 auth = { key = 1, password = "password" } # Default YubiHSM admin credentials. Change ASAP!
 keys = [{ id = "gaia-9000", key = 1 }]
 #serial_number = "0123456789" # identify serial number of a specific YubiHSM to connect to
+
+[[providers.ledgertm]]
```

Comment:
Is this everything that is necessary to get the ledger up and running? If not providing (all) existing options with some exemplary defaults would be cool.